src/server/server.js

```diff
@@ -72,7 +72,7 @@ app.get('/journalism/teams/:teamname', (req, res) => {
   journalismTeam(res, req.params.teamname);
 });
 
-app.get('/journalism/player/:playername', (req, res) => {
+app.get('/journalism/players/:playername', (req, res) => {
```

Comment:
would rather not change, cause there are no other changes, at least it's consistent now X) 
I mean my thinking was that we're selecting a single player from players and likewise with teams